src/GrammarDecl.js

```diff
@@ -21,6 +22,10 @@ function GrammarDecl(name) {
 
 // Helpers
 
+GrammarDecl.prototype.at = function(startIdx, endIdx) {
```

Comment:
I've changed it as you suggested. `at` was deliberately chosen to be short so as to not clutter the recipes -- `.withInterval(decl.sourceInterval())` is a lot of noise when it appears on every single expression. It's not a method that should really appear anywhere other than recipes, so I thought it would be better to make it short rather than descriptive.

But, it looks like we are going to revisit the recipes soon anyways, so I don't really care too much.